app/views/dashboard/giveaways/_claimed_by.html.erb

```diff
@@ -0,0 +1,17 @@
+<% unless @giveaway.claims.empty?%>
+  <% if @giveaway.approved_to.nil?%>
```

Comment:
Can we stored approved giveaways in an instance variable in the controller instead of checking for two conditions here?

Reply:
@violeta-p Even with two instance variables in the controller I need both conditions.
First if the giveaway has a Claim.
Next what's its 'approved_to' attribute status? Depending on it's status I need another, second condition to render different result. I don't understand how you achieve that with two different variables and without a condition on which one to render? Please advice.

Reply:
Okay then, we can go with two conditions.